eth/tracers/native/prestate.go

```diff
@@ -179,14 +156,38 @@ func (t *prestateTracer) CaptureState(pc uint64, op vm.OpCode, gas, cost uint64,
 	}
 }
 
-func (t *prestateTracer) CaptureTxStart(gasLimit uint64) {
-	t.gasLimit = gasLimit
+func (t *prestateTracer) CaptureTxStart(env *vm.EVM, tx *types.Transaction) {
```

Comment:
I'm wondering if it's possible that between txStart and Start, it directly exited due to an error, so the old version of prestate was not able to capture any data, while now it can. Compared to before, this is an improvement.